### PR TITLE
remove duplicate contributor profiles

### PR DESCRIPTION
**Pull request by mikeattara**
I removed duplicate profiles for Matt and Ankur, and also restructure list

### CODE REVIEW

**Comment by The24thDS** [comment]
Just wanted to help you but you figured it out ;)

**Comment by mikeattara** [comment]
@The24thDS  I was trying to restructure the code. I feel `li` must rather be the direct child of `ul` and have the `a`  as a child instead on the current code

**Comment by mikeattara** [comment]
@MattCSmith and @aneagoie  What do you think?

**Comment by The24thDS** [comment]
You're correct about that, but the problem you had before was because you had two `li` end tags without an open one

**Comment by mikeattara** [comment]
Aww, I missed that

File: src/components/Footer/Footer.js [hunk 1]
```diff
@@ -42,16 +42,17 @@ const Footer = ({ contrib }) => (
...
-              <Contrib contributors={contrib} />
+                <li className="grow">
+                  <a href="https://github.com/Dhaval1403" 
+                    target="_blank" 
+                    rel="noopener noreferrer"
+                  >
+                    Dhaval Mehta
+                  </a>
+                </li>
+                </li className="grow">
```
Comment (The24thDS) [inline src/components/Footer/Footer.js]: Over here @mikeattara

**Pull request merged**